acceptance/api_v1_serviceplans_test.go

```diff
@@ -0,0 +1,76 @@
+package acceptance_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+
+	"github.com/epinio/epinio/internal/services"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("ServicePlans API Application Endpoints", func() {
+
+	var org string
+
+	BeforeEach(func() {
+		org = newOrgName()
+		setupAndTargetOrg(org)
+
+		// Wait for server to be up and running
+		Eventually(func() error {
+			_, err := Curl("GET", serverURL+"/api/v1/info", strings.NewReader(""))
+			return err
+		}, "1m").ShouldNot(HaveOccurred())
+	})
+
+	Context("ServicePlans", func() {
+
+		BeforeEach(func() {
+			setupInClusterServices()
+		})
+
+		Describe("GET /api/v1/orgs/:org/serviceclasses/:serviceclass/serviceplans", func() {
+			var servicePlanNames []string
+			var servicePlanDescs []string
+			var servicePlanFrees []bool
+
+			It("lists all serviceplans in the org", func() {
+				response, err := Curl("GET", fmt.Sprintf("%s/api/v1/orgs/%s/serviceclasses/%s/serviceplans",
+					serverURL, org, "mariadb"), strings.NewReader(""))
+				Expect(err).ToNot(HaveOccurred())
+				Expect(response).ToNot(BeNil())
+				defer response.Body.Close()
+				bodyBytes, err := ioutil.ReadAll(response.Body)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(response.StatusCode).To(Equal(http.StatusOK), string(bodyBytes))
+
+				var data services.ServicePlanList
+				err = json.Unmarshal(bodyBytes, &data)
+				Expect(err).ToNot(HaveOccurred())
+				for _, servicePlan := range data {
+					servicePlanNames = append(servicePlanNames, servicePlan.Name)
+					servicePlanDescs = append(servicePlanDescs, servicePlan.Description)
+					servicePlanFrees = append(servicePlanFrees, servicePlan.Free)
+				}
+				Expect(servicePlanNames).Should(ContainElements("10-1-26", "10-1-28", "10-3-20", "10-3-16", "10-3-17"))
+				Expect(servicePlanDescs).Should(ContainElements("Fast, reliable, scalable, and easy to use open-source relational database system. MariaDB Server is intended for mission-critical, heavy-load production systems as well as for embedding into mass-deployed software. Highly available MariaDB cluster."))
+				Expect(servicePlanFrees).Should(ContainElements(true, true))
+			})
+
+			It("returns a 404 when the org does not exist", func() {
+				response, err := Curl("GET", fmt.Sprintf("%s/api/v1/orgs/:org/idontexist/:serviceclass/serviceplans", serverURL), strings.NewReader(""))
```

Comment:
Doesn't that URI need to match the URL pattern from the router?

```suggestion
				response, err := Curl("GET", fmt.Sprintf("%s/api/v1/orgs/idontexist/serviceclasses/someclass/serviceplans", serverURL), strings.NewReader(""))
```

Nit: not a big fan of the existing `Curl` func, it's not really curl and we could get rid of a param by having `Get()`, `Post()` etc. 
Also while you can send a request body with a GET requests, it's not often (never) used. So that param is also useless. (https://stackoverflow.com/a/978094)

Reply:
That might be a good suggestion. I will create a new issue for this.